code/game/gamemodes/miniantags/abduction/gland.dm

```diff
@@ -13,7 +13,6 @@
 	var/human_only = 0
 	var/active = 0
 	tough = TRUE //not easily broken by combat damage
-	sterile = TRUE //not very germy
```

Comment:
No change in behavior; now that everything is properly unified in terms of robotic/ORGAN_ROBOT, `sterile` is only needed for *organic* organs that need to not be impacted by germs.